templates/data/serverless.yaml

```diff
@@ -116,6 +126,133 @@ templates:
                     - jobConfig_postsubmit
                   local:
                     - skip_if_only_changed_documentation
+                    - branches_legacy
+      - to: ../../prow/jobs/kyma-project/serverless/serverless-build.yaml
+        localSets:
+          skip_if_only_changed_documentation:
+            skip_if_only_changed: '^docs/|^examples/'
+          default:
+            branches:
+              - main
+              - release-1.3
+              - release-1.4
```

Comment:
I would suggest to add `release-0.0` branch, because it's branch we use to test release pipeline